### PR TITLE
[IIIF-418] Bundle Test, Prod, Sinai environments

### PR DESCRIPTION
**Pull request by cachemeoutside**
This is a pretty big change, but here's the summary:
* Revamped test-iiif, iiif, and sinaimanuscripts to use external written modules stored in github repository. Clean up of old static modules is also included in this PR.
* Bundled Manifeststore and Cantaloupe to run within the same container host, but on different ports to save resource costs. This means each environment has an instance of manifeststore running on it
* Each environment essentially has the same template across the board, with a local.secrets file is kept and ran locally on an individual's workstation
* Configured each environment to use cantaloupe-ucla:latest until a stable fix is released
* This is already ran and deployed onto each environment with success
* I will create a tag after this PR is merged

### CODE REVIEW

File: environments/prod-iiif/bin/destroy.sh [hunk 1]
```diff
@@ -4,7 +4,7 @@ BACKEND_FILE="backend.hcl"
 TERRAFORM="/opt/terraform/bin/terraform"
 PLAN_FILE="current.plan"
 LOCAL_SECRETS="local.secrets"
-WORKSPACE="prod"
+WORKSPACE="test"
```
Comment (ksclarke) [inline environments/prod-iiif/bin/destroy.sh]: This file is still in a `environment/prod-iiif` dir? Is this intended? The text is being changed from `prod` to `test`

File: environments/prod-iiif/README.md [hunk 2]
```diff
@@ -9,7 +9,7 @@ This environment heavily utilizes modules to build AWS environments via Terrafor
 After editing the vars file, you can run the bootstrap script
 
 ```
-cd environments/prod
```
Comment (ksclarke) [inline environments/prod-iiif/README.md]: This file is still in a `environment/prod-iiif` dir? Is this intended? The text is being changed from `prod` to `test`

File: environments/prod-sinai/README.md [hunk 3]
```diff
@@ -0,0 +1,25 @@
+# Test Environment
```
Comment (ksclarke) [inline environments/prod-sinai/README.md]: Again, unsure why a test environment readme is in a `environments/prod-sinai` dir(?)

File: environments/prod-sinai/bin/destroy.sh [hunk 4]
```diff
@@ -1,32 +1,33 @@
 #!/bin/bash
 
 BACKEND_FILE="backend.hcl"
+TERRAFORM="/opt/terraform/bin/terraform"
 PLAN_FILE="current.plan"
 LOCAL_SECRETS="local.secrets"
-WORKSPACE="sinai-iiif"
+WORKSPACE="test"
```
Comment (ksclarke) [inline environments/prod-sinai/bin/destroy.sh]: test workspace inside an `environments/prod-sinai` dir?

**Comment by cachemeoutside** [comment]
Good catch. I don't ever really use the destroy script in these environments. I've consolidated this to a run script where a destroy can be triggered by a flag. Also updated a README to be placed at the root path.

**Pull request merged**